src/index.html

```diff
@@ -9,6 +9,50 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="header-wrapper">
+        <img
+        src="logo.png"
+        class="logo"
+        alt="Logo"
+        >
```

Comment:
```suggestion
        <img
          src="logo.png"
          class="logo"
          alt="Logo"
        >
```
Consider fixing indents according to the code style guide